installer.sh

```diff
@@ -117,7 +118,7 @@ while getopts ":shotkmp:" opt; do
             echo $'-k \t\t\t\t Download Keylime (stub installer mode)'
             echo $'-o \t\t\t\t Use OpenSSL instead of CFSSL'
             echo $'-t \t\t\t\t Create tarball with keylime_agent'
-            echo $'-m \t\t\t\t Use modern TPM 2.0 libraries (vs. TPM 1.2)'
+            echo $'-m \t\t\t\t Use legacy TPM version 1.2'
```

Comment:
The logic is inverted here now, so this will do the opposite of what is listed.  Also, the "-m" flag no longer makes sense (modern vs. legacy).

Reply:
Sorry, that's a long day mistake from me.

I was going to change the default to being tpm2.0, but then thought it better for another patch so we can discuss it first. I will revert that change.